docs/source/markdown/podman-run.1.md

```diff
@@ -609,6 +609,10 @@ If a container is run with a pod, and the pod has an infra-container, the infra-
 
 Run container in an existing pod and read the pod's ID from the specified file.  If a container is run within a pod, and the pod has an infra-container, the infra-container will be started before the container is.
 
+**--preserve-fds**=*N*
+
+Pass down to the process N additional file descriptors (in addition to 0, 1, 2).  The total FDs will be 3+N.
```

Comment:
Mention this is local-only

Reply:
Actually, maybe not... I don't think we do this for --latest, do we?

Reply:
> Actually, maybe not... I don't think we do this for --latest, do we?
>

 I don't see podman run has `--latest`. podman exec with both --latest and --preserve-fds seems working
```
$ podman exec -it --latest --preserve-fds=1 sh
/ # ls -l /proc/self/fd
total 0
lrwx------    1 root     root            64 Jun 16 19:52 0 -> /dev/pts/0
lrwx------    1 root     root            64 Jun 16 19:52 1 -> /dev/pts/0
lrwx------    1 root     root            64 Jun 16 19:52 2 -> /dev/pts/0
lrwx------    1 root     root            64 Jun 16 19:52 3 -> /null
ls: /proc/self/fd/4: cannot read link: No such file or directory
lr-x------    1 root     root            64 Jun 16 19:52 4
```


> Mention this is local-only

the `--preserve-fds` was hidden if remote in `podman exec`, so I also hide it for `podman run`. I can mention it is local-only.

Reply:
I don't think we need further manpage changes, it looks like other flags hidden if remote are still in the manpages.